core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-device-selector-seo/dot-device-selector-seo.component.scss

```diff
@@ -0,0 +1,116 @@
+@use "variables" as *;
+@import "dotcms-theme/utils/theme-variables";
+@import "dotcms-theme/utils/extends";
+
+.dotDeviceSelector__panel {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    border-radius: $border-radius-md;
+    border: 1px solid $color-palette-gray-300;
+}
+
+.dotDeviceSelector__header {
+    display: flex;
+    height: 2.5rem;
+    padding: $spacing-1 $spacing-3;
+    justify-content: space-between;
+    align-items: center;
+    align-self: stretch;
+
+    background: $color-palette-gray-200;
+}
+
+.dotDeviceSelector__header-title {
+    font-size: 1.25rem;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 140%;
+}
+
+.dotDeviceSelector__column {
+    gap: 1rem;
+    display: flex;
+    flex-direction: column;
+}
+
+.dotDeviceSelector__content {
+    display: flex;
+    .device-list {
+        width: 100%;
+        padding: 0;
+        .device-list__item {
+            display: flex;
+            list-style: none;
+            width: 15.75rem;
+            height: 3.625rem;
+            padding: $spacing-1 $spacing-3 $spacing-1 $spacing-3;
+            gap: 0.5rem;
+            cursor: pointer;
+
+            .device-list__item__content {
+                display: flex;
+                flex-direction: column;
+            }
+
+            .device-list__item__dimensions {
+                color: $color-palette-gray-600;
+            }
+
+            .device-list__item__name {
+                line-height: 1;
+            }
+
+            &:hover {
+                background-color: $color-palette-gray-100;
+            }
+        }
+
+        .device-list__item.device-list__item-media {
+            width: 15.75rem;
+            height: 2.5rem;
+        }
+    }
+}
+
+.dotDeviceSelector__link--container {
+    display: flex;
+    cursor: pointer;
+    flex-grow: 3;
```

Comment:
Why is this?

Reply:
![image](https://github.com/dotCMS/core/assets/3438705/65b8186b-6cba-431e-9125-f492b81f8419)